components/email-mgt/org.wso2.carbon.email.mgt/src/main/java/org/wso2/carbon/email/mgt/constants/I18nMgtConstants.java

```diff
@@ -37,8 +37,8 @@ private I18nMgtConstants() {}
     public static final String SMS_CONF_DIRECTORY = "sms";
     public static final String EMAIL_ADMIN_CONF_FILE = "email-admin-config.xml";
     public static final String SMS_TEMPLAE_ADMIN_CONF_FILE = "sms-templates-admin-config.xml";
-    public static final String DEFAULT_EMAIL_LOCALE = "en_us";
-    public static final String DEFAULT_SMS_NOTIFICATION_LOCALE = "en_us";
+    public static final String DEFAULT_EMAIL_LOCALE = "en_US";
```

Comment:
Will there be any impact in changing these constants?

Reply:
This change no longer needed with new impl: https://github.com/wso2-extensions/identity-event-handler-notification/pull/230